docs/Installation.md

```diff
@@ -39,6 +39,7 @@ Make sure that Elasticsearch is started on boot:
     /bin/systemctl daemon-reload
     /bin/systemctl enable elasticsearch.service
     /bin/systemctl start elasticsearch.service
+    /bin/systemctl status elasticsearch.service
```

Comment:
Maybe add something like:

### Make sure Elasticsearch is running correctly
/bin/systemctl status elasticsearch.service